dnf5.spec

```diff
@@ -254,6 +262,8 @@ It supports RPM packages, modulemd modules, and comps groups & environments.
 %dir %{_datadir}/dnf5
 %dir %{_datadir}/dnf5/aliases.d
 %config %{_datadir}/dnf5/aliases.d/compatibility.conf
+%config %{_unitdir}/dnf5-offline-transaction.service
```

Comment:
Uhh, why is something in `/usr` marked as a config file?!

Reply:
> I was wondering why the offline execution doesn't work for me when I installed dnf5 from the PR build.

Ah, I had been testing mostly with `make install` and not building the RPMs.

> I think we also need %{_unitdir}/system-update.target.wants/dnf5-offline-transaction.service to enable the service.
I am not sure why the rpm build passed because the symlink is INSTALLed in cmake and I would think its an extra file that is missing in the spec.
Maybe the symlinks are special and we will need something like: https://github.com/rpm-software-management/dnf-plugins-core/blob/master/dnf-plugins-core.spec#L494-L497?

I tried for a bit to have CMake create the symlink and get RPM to package it correctly, but I couldn't get it working, so I've just done it in the spec file, like in dnf-plugins-core.spec as you mentioned. The CMake script doesn't install the service file to `system-update.target.wants`, which is unfortunate since it won't get installed when you do `make install`, but at least the RPM package works now..

> I am also thinking if we could check that the service is present and enabled before doing the reboot?

Good idea, done.